nautobot/core/views/generic.py

```diff
@@ -1041,7 +1041,7 @@ def post(self, request, **kwargs):
                 try:
                     with deferred_change_logging_for_bulk_operation():
                         updated_objects = []
-                        for obj in self.queryset.filter(pk__in=form.cleaned_data["pk"]):
+                        for obj in queryset:
```

Comment:
Any significance to the fact that we were doing `pk__in=form.cleaned_data["pk"]` before but now (lines 1023-1024 above) we're doing `pk__in=request.POST.getlist("pk")`? Any subtle behavior change that results from this?

Reply:
My concern was about the non-`edit_all` case - just trying to understand the significance of the differing logic between line 1044 (before) and line 1024 (after).